README.rst

```diff
@@ -122,6 +122,66 @@ For example, a custom drill-down that offers only past dates:
                     ) if day <= today
                 )
 
+
+EnhancedDateHierarchy
+----------------------------------------------
+
+Django filter the query set for a given level in the date hierarchy using a database
+function to extract the relevent date part. For example, when filtering a queryset on
+a `created` date field for November 2017 Django will execute the following query:
+
+.. code-block:: sql
+
+    SELECT
+        ...
+    FROM
+        app_table
+    WHERE
+        created BETWEEN '2017-01-01 00:00:00' AND '2017-12-31 23:59:59.999999'
+        AND EXTRACT('month', created) = 11
+
+Using a function to filter the queryset often leads to poor performance. The database
+optimizer cannot always estimate the cardinality of such condition which might lead to
```

Comment:
such condition -> such a condition

Reply:
I don't think estimating cardinality is really the issue here. The main problem is that a function is opaque to the database optimizer (at least in postgres). Therefore, if you have a range-based (btree) index on the field, using `EXTRACT` does not limit the range at all, and so the index is not utilized properly.